curriculum/challenges/english/25-front-end-development/quiz-relational-database/66f1af82732957c895f0b21a.md

```diff
@@ -17,439 +17,438 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is a relational database?
 
 #### --distractors--
 
-Placeholder distractor 1
+A database structured to recognize non-relation between stored items
```

Comment:
```suggestion
A database structured to recognize non-relation between stored items.
```

when answers are full sentences they also need the full stop

Reply:
Should I add a full stop in options too?

Reply:
yes, all answers/distractors that are full sentences

Reply:
Please check now!